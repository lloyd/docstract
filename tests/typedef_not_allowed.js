/** Simple typedef usage */

/**
 * @typedef object
 */

/**
 * @prop foo {string}
 */

/**
 * @endtypedef
 */
