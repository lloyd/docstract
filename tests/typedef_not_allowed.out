{
  "args": [
    "@typedef not allowed in global context at line 3"
  ], 
  "exception_type": "<type 'exceptions.RuntimeError'>", 
  "stack": [
    [
      "<module>", 
      "got = ds.extractFromFile(os.path.join(testDir, test + \".js\"))"
    ], 
    [
      "extractFromFile", 
      "data = self.extract(contents)"
    ], 
    [
      "extract", 
      "self._analyzeBlock(block, context, firstBlock, stack, lineStart, line)"
    ], 
    [
      "_analyzeBlock", 
      "thisContext))"
    ]
  ]
}
